packages/components/src/date-time/index.js

```diff
@@ -1,24 +1,30 @@
 /**
  * Internal dependencies
  */
+import 'react-dates/initialize';
```

Comment:
What does this code do? It looks like it has some kind of side-effects. Shouldn't it be applied on the application level?

Reply:
Seems like, but if we don't initialise it here, won't this package–on its own–be useless/unstable?

Reply:
Also: I'm not clear where we'd want to move it to? The lerna package system muddies things like this for me... would this go in the `components/src/index.js` file? Or somewhere in `edit-post/index.js`? 🤷‍♂️

Reply:
I tried moving this out of `packages/components/src/date-time/index.js` and wound up with errors. This seems fine to leave here 🤷‍♂️